### ISSUE TITLE
Consolidate errors messages on from fields

### ISSUE THREAD

**Issue by bwulff** [slidewiki/slidewiki-platform#1202]

**Comment by aoelen** [comment]
More detailed description: 
Form errors should be more consistent across SlideWiki. Additionally, errors on form fields should be accessible to screen readers. On submit, focus should go to the first form element with an error. 

The following design should be implemented across the platform:
![image](https://user-images.githubusercontent.com/1567948/67665228-9b115f00-f969-11e9-9613-b5b40db02c3e.png)

The following forms should be changed:
- Login
- Registration
- Add deck (design is in place, make error messages accessible) 
- Alter my personal data
- Change password
- Deck settings
- Add source
- Add comment 
- Create a new playlist
